test/browser-tests/window-utils/serviceworkers/immediate-control.js

```diff
@@ -0,0 +1,2 @@
+self.addEventListener('install', () => self.skipWaiting());
```

Comment:
I'm not sure what the ambiguity is, though? AFAIK there are no browsers that ever required an `event.waitUntil()` wrapper around `self.skipWaiting()` or `self.clients.claim()`.

There are old examples that used `event.waitUntil()` (including many I wrote), but it turns out it was never actually necessary.